src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_validators.py

```diff
@@ -429,7 +429,7 @@ def _validate_vm_create_availability_set(namespace):
             name=name)
 
 
-def _validate_vm_create_vnet(namespace, for_scale_set=False):
+def _validate_create_vnet(namespace, for_scale_set=False):
```

Comment:
Why the name change here? The old name made it clear you wouldn't use this for just validating any old VNET.